src/redux/reducers/lightReducer.js

```diff
@@ -0,0 +1,16 @@
+const initial = {
+    light: 'light'
+}
+
+function lightReducer(state = initial, action) {
+    switch(action.type) {
+        case 'CHANGE_LIGHT':
+            return {
+                light: document.body.classList.toggle('dark')
```

Comment:
в редьюсере не должно быть никаких ссылок на document, там надо хранить только значение 
```suggestion
                light: true
```